apps/condo/domains/marketplace/hooks/useMarketplaceServicesTableColumns.tsx

```diff
@@ -0,0 +1,104 @@
+    const SkuTitle = intl.formatMessage({ id: 'pages.condo.marketplace.services.table.sku' })
+    const SkuNameTitle = intl.formatMessage({ id: 'pages.condo.marketplace.services.table.name' })
+    const CategoryTitle = intl.formatMessage({ id: 'pages.condo.marketplace.services.table.category' })
+    const ScopeTitle = intl.formatMessage({ id: 'pages.condo.marketplace.services.table.addressesAndPrices' })
+
+    const router = useRouter()
+    const { filters, sorters } = parseQuery(router.query)
+    const sorterMap = getSorterMap(sorters)
+
+    const search = getFilteredValue(filters, 'search')
+    const render = useMemo(() => getTableCellRenderer({ search }), [search])
+
+    const processedScopes = {}
+    for (const scope of marketPriceScopes) {
+        const { marketItemPrice: { marketItem: { id }, price }, property: { addressMeta: { data: { street_with_type, house_type, house } } }  } = scope
+
+        const item = {
+            price: get(price[0], 'price'), 
+            currency: get(price, 'currencyCode', 'RUB'), 
+            address: `(${street_with_type}, ${house_type}. ${house})`,
+        }
+        
+        if (Array.isArray(processedScopes[id])) processedScopes[id] = [...processedScopes[id], item]
+        processedScopes[id] = [item]
+    }
+
+    return useMemo(() => {
+        return [
+            {
+                title: SkuTitle,
+                sortOrder: get(sorterMap, 'sku'),
+                filteredValue: getFilteredValue(filters, 'sku'),
+                key: 'sku',
+                dataIndex: 'sku',
+                sorter: true,
+                width: '10%',
+                render: render,
+                filterDropdown: getFilterDropdownByKey(filterMetas, 'sku'),
+            },
+            {
+                title: SkuNameTitle,
+                filteredValue: getFilteredValue(filters, 'name'),
+                key: 'name',
+                dataIndex: 'name',
+                width: '23%',
+                filterDropdown: getFilterDropdownByKey(filterMetas, 'name'),
+                render: render,
+                filterIcon: getFilterIcon,
+            },
+            {
+                title: CategoryTitle,
+                filteredValue: getFilteredValue(filters, 'category'),
+                key: 'category',
+                dataIndex: 'marketCategory',
+                width: '23%',
+                filterDropdown: getFilterDropdownByKey(filterMetas, 'category'),
+                render: (category) => {
+                    const categoryName = intl.formatMessage( { id: `marketplace.marketCategory.${category.name}.name` })
+
+                    if (category.parentCategory) {
+                        const parentCategoryName = intl.formatMessage( { id: `marketplace.marketCategory.${category.parentCategory.name}.name` })
+                        return render(`${parentCategoryName}»${categoryName}`)
+                    }
+                    return render(categoryName)
+                },
```

Comment:
We also need to show only `parentCategoryName` if only a single subcategory is within.